docs/scos/dev/feature-integration-guides/202204.0/mailing-and-notifications-feature-integration.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Mailing and notifications feature integration
+description: The guide describes the process of installing the mailing provider in your project.
+last_updated: Nov 21, 2022
+template: feature-integration-guide-template
+---
+
+{% info_block errorBox %}
+
+The following feature integration guide expects the basic feature to be in place.
```

Comment:
`Mail and notifications feature`, the one that the document belongs to.
This message is added, because IG is not complete, and contains only parts related to `SymfonyMailer`

Reply:
got it, makes sense as is then